playground/package.json

```diff
@@ -5,6 +5,7 @@
   "private": true,
   "scripts": {
     "build": "mc-scripts build",
+    "start:server": "NODE_ENV=development && ts-node ./src/server.ts",
```

Comment:
FYI: The playground uses the Vercel serverless functions (`./api/echo.ts`), so we can simply use [vercel dev](https://vercel.com/docs/cli#commands/dev) command.

Have you checked that to see if it works?

Reply:
I checked but it seems we need to have a project defined in Vercel linked to the one you want to test locally.
When using that package it requires you to link or create a new project in Vercel.
If we can have a permanent project for this, we might use that package.

Reply:
Ok, I double checked that and it seems the process now is a bit different than it was before.

Let's keep our custom script then, sorry about that.